.markdownlint-cli2.jsonc

```diff
@@ -0,0 +1,14 @@
+// https://github.com/DavidAnson/markdownlint-cli2#markdownlint-cli2jsonc
+{
+  "config": {
+    "MD013": false,
+    "MD024": false,
+    "MD033": false,
+    "MD041": false,
```

Comment:
This one also makes sense to me:

[MD041](https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md041.md) first-line-heading/first-line-h1 - First line in a file should be a top-level heading

Should we keep it disabled? now or eventually

Reply:
Same answer as above: out of scope for this PR.

FWIW, here are the current failures:

```
.github/ISSUE_TEMPLATE/discussion.md:11 MD041/first-line-heading/first-line-h1 First line in a file should be a top-level heading [Context: "- [ ] This only affects the Ja..."]
.github/ISSUE_TEMPLATE/feature_request.md:7 MD041/first-line-heading/first-line-h1 First line in a file should be a top-level heading [Context: "**NB:** Before opening a featu..."]
.github/ISSUE_TEMPLATE/request_for_instrumentation.md:13 MD041/first-line-heading/first-line-h1 First line in a file should be a top-level heading [Context: "### Is it applicable for Node ..."]
.github/PULL_REQUEST_TEMPLATE.md:16 MD041/first-line-heading/first-line-h1 First line in a file should be a top-level heading [Context: "## Which problem is this PR so..."]
experimental/examples/logs/README.md:1 MD041/first-line-heading/first-line-h1 First line in a file should be a top-level heading [Context: "## Installation"]
README.md:2 MD041/first-line-heading/first-line-h1 First line in a file should be a top-level heading [Context: "---"]
selenium-tests/README.md:1 MD041/first-line-heading/first-line-h1 First line in a file should be a top-level heading [Context: "## Selenium Tests"]
```